docarray/array/doc_vec/doc_vec.py

```diff
@@ -83,6 +84,15 @@ class DocVec(AnyDocArray[T_doc]):
     this Document "view" is similar to the behavior of `view = tensor[i]` in
     numpy/PyTorch.
 
+    !!! note
```

Comment:
Maybe explain that otherwise it cannot be stacked, otherwise this seems a bit arbitrary